e2ev2/tests/workspace-create.test.js

```diff
@@ -0,0 +1,70 @@
+const fsp = require('fs').promises
```

Comment:
Is this a close as possible copy of an existing test into the new testing approach? If so, what would be the best way to handle duplicative tests?